AdyenComponents/PayTo/PayToComponent.swift

```diff
@@ -11,6 +11,12 @@ import UIKit
 public final class PayToComponent: PaymentComponent,
                                    PresentableComponent {
 
+    private enum ViewIdentifier {
+        static let flowSelectionTitleLabelItem = "flowSelectionTitleLabelItem"
+        static let flowSelectionItem = "flowSelectionSegmentedControlItem"
+        static let continueButtonItem = "continueButton"
```

Comment:
It's a bit inconsistent, let's remove Item from prior two or ad "item" to the button identifier? If its main purpose it accessibility ID, I suggest removeing "Item" from all string identifiers as it doesn't bring any value on accessibility layer.

Reply:
Removed "item" from string identifier ce0a1a60db9fc6ca39b38da911d6df89429afec1